src/components/ReactTable/ReactTable.tsx

```diff
@@ -6,10 +6,18 @@ import { Column, TableState, useSortBy, useTable } from 'react-table'
 type ReactTableProps = {
   columns: Column<any>[]
   data: object[]
+  displayHeaders?: boolean
+  onRowClick?: () => void | undefined
```

Comment:
Is there ever a case where `undefined` will be explicitly returned? ref: https://github.com/shapeshift/web/pull/1329/files#diff-96dc327eb19b724e0fcbc22953cd63b9fe03c21cc27ac816ddb6024df6c6b658R92